runtime/parachains/src/hrmp.rs

```diff
@@ -908,6 +908,19 @@ impl<T: Config> Pallet<T> {
 		weight
 	}
 
+	/// Worst case weight for `prune_hrmp`.
+	pub(crate) fn prune_hrmp_weight(
```

Comment:
I think if we have #3876 already in, we can automatically measure this function.